scala-diff/shared/src/main/scala/munit/diff/Clue.scala

```diff
@@ -0,0 +1,13 @@
+package munit.diff
+
+class Clue[+T](
```

Comment:
I don't think `Clue` should be a type in the diff library.  From what I understand, it's been moved here so that we can give it a custom representation in `Printers`.

That being said, I haven't thought of alternative approaches.

Reply:
Me neither, I did this to satisfy the compiler, I also don't have a better idea yet